stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.0
+version: 5.0.1
```

Comment:
Could you change again to 5.0.2 for now please?

Reply:
Done